cmd/govim/internal/types/point.go

```diff
@@ -26,66 +26,57 @@ type Point struct {
 }
 
 func PointFromOffset(b *Buffer, offset int) (Point, error) {
-	tf := b.tokFile()
-	line, col, err := span.ToPosition(tf, offset)
+	m := protocol.NewMapper(b.URI(), b.contents)
```

Comment:
My only concern here (and below) would be that we create a new mapper every time we call this function. Previously, there was lazy creation and then reuse, no?

Reply:
You are absolutely right, I fixed it.